testscommon/utils.go

```diff
@@ -13,9 +13,3 @@ func AddTimestampSuffix(tag string) string {
 	timestamp := time.Now().Format("20060102150405")
 	return fmt.Sprintf("%s_%s", tag, timestamp)
 }
-
-func panicIfError(message string, err error) {
```

Comment:
why is it better to move it in a random file from package?

Reply:
because it was used only there